<!-- BEGIN KEY -->
public final class org.omg.CORBA.UnknownUserException extends org.omg.CORBA.UserException
<!-- END KEY -->
<!-- BEGIN DATA -->
包含由服务器返回的用户异常的类。当客户端使用 DII 执行调用时，从服务器返回的任何异常都被封闭在 <code>UnknownUserException</code> 对象所包含的 <code>Any</code> 对象中。此异常可从 <code>Request.env</code> 方法返回的 <code>Environment</code> 对象中得到。

@see <A href="../../../../technotes/guides/idl/jidlExceptions.html">关于 Java&#xA0;IDL 异常的文档</A>
@see Request

<!-- END DATA -->
<!-- BEGIN KEY -->
public org.omg.CORBA.Any org.omg.CORBA.UnknownUserException.except
<!-- END KEY -->
<!-- BEGIN DATA -->
包含由服务器抛出的实际用户异常的 <code>Any</code> 实例。
@serial

<!-- END DATA -->
<!-- BEGIN KEY -->
public org.omg.CORBA.UnknownUserException()
<!-- END KEY -->
<!-- BEGIN DATA -->
构造一个 <code>UnknownUserException</code> 对象。

<!-- END DATA -->
<!-- BEGIN KEY -->
public org.omg.CORBA.UnknownUserException(Any a)
<!-- END KEY -->
<!-- BEGIN DATA -->
构造一个包含给定 <code>Any</code> 对象的 <code>UnknownUserException</code> 对象。

@param a 包含服务器返回的用户异常的 <code>Any</code> 对象

<!-- END DATA -->
